flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksdbStateFileVerifierTest.java

```diff
@@ -0,0 +1,124 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.contrib.streaming.state;
+
+import org.apache.flink.util.FileUtils;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.rocksdb.Checkpoint;
+import org.rocksdb.ColumnFamilyDescriptor;
+import org.rocksdb.ColumnFamilyOptions;
+import org.rocksdb.DBOptions;
+import org.rocksdb.Options;
+import org.rocksdb.RocksDB;
+import org.rocksdb.WriteOptions;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.flink.contrib.streaming.state.snapshot.RocksSnapshotUtil.SST_FILE_SUFFIX;
+import static org.junit.Assert.fail;
+
+/** {@link RocksDBStateFileVerifier} test. */
+public class RocksdbStateFileVerifierTest {
+
+    @TempDir java.nio.file.Path folder;
+
+    @Test
+    public void rocksdbStateFileVerifierTest() throws Exception {
+        List columnFamilyHandles = new ArrayList<>(1);
+        String rootPath = folder.toAbsolutePath().toString();
+        File dbPath = new File(rootPath, "db");
+        File cpPath = new File(rootPath, "cp");
+
+        try (DBOptions dbOptions = new DBOptions().setCreateIfMissing(true);
+                ColumnFamilyOptions colOptions = new ColumnFamilyOptions();
+                Options sstFileReaderOptions = new Options(dbOptions, colOptions);
+                WriteOptions writeOptions = new WriteOptions().setDisableWAL(true);
+                RocksDB db =
+                        RocksDB.open(
+                                dbOptions,
+                                dbPath.toString(),
+                                Collections.singletonList(
+                                        new ColumnFamilyDescriptor(
+                                                "default".getBytes(), colOptions)),
+                                columnFamilyHandles);
+                RocksDBStateFileVerifier rocksDBStateFileVerifier =
+                        new RocksDBStateFileVerifier(sstFileReaderOptions)) {
+
+            byte[] key = "checkpoint".getBytes();
+            byte[] val = "incrementalTest".getBytes();
+            db.put(writeOptions, key, val);
+
+            try (Checkpoint checkpoint = Checkpoint.create(db)) {
+                checkpoint.createCheckpoint(cpPath.toString());
+            }
+
+            List<Path> sstFiles =
+                    Arrays.stream(FileUtils.listDirectory(cpPath.toPath()))
+                            .filter(file -> file.getFileName().toString().endsWith(SST_FILE_SUFFIX))
+                            .collect(Collectors.toList());
+
+            Assert.assertFalse(sstFiles.isEmpty());
```

Comment:
```suggestion
            assertThat(sstFiles).isNotEmpty();
```

It's from `org.assertj.core.api.Assertions#assertThat`